src/warnet/lnnode.py

```diff
@@ -99,7 +106,8 @@ def export(self, config, subdir):
         config["nodes"].append(
             {
                 "id": container_name,
-                "address": f"https://{self.ipv4}:{self.rpc_port}",
+                # TODO: What does this break?
+                # "address": f"https://{self.ipv4}:{self.rpc_port}",
```

Comment:
Great point actually. So, running warnet+simln locally, simln would be running on the host and not in the docker network at all. It would use the IPs of the lnd nodes inside docker to send RPC commands (remember we made them fix it, https://github.com/bitcoin-dev-project/sim-ln/pull/151).

This may be the only time we explicitly need the actual IP of the lnd node.

Or we could work out how to incorporate sim-ln inside a container in the network, which we will probably need for K8s deployment anyway